src/WebJobs.Extensions.Http/Routing/WebJobsRouter.cs

```diff
@@ -51,6 +51,17 @@ private void InitializeRouteCollections()
 
         public Task RouteAsync(RouteContext context)
         {
+            // /admin/* routes should not be allowed to be overriden by proxies or function routes.
+            var path = context.HttpContext.Request.Path;
+            if (path.StartsWithSegments(new PathString("/admin"), System.StringComparison.OrdinalIgnoreCase))
+            {
+                // admin/warmup is handled by function routes
+                if (!path.StartsWithSegments(new PathString("/admin/warmup"), System.StringComparison.OrdinalIgnoreCase))
+                {
+                    return Task.CompletedTask;
```

Comment:
Just to clarify, if we return a `Task.CompletedTask`, presumably that means that some other router further down the pipeline will handle the request (or if none do, then we return a 404?).

Is there a reason we went with this approach instead of having the Admin controller get routing priority over HTTP triggers?

Also, this only bypasses admin requests, but what about extension webhook endpoints? I know both EventGrid and Durable Functions use the webhook endpoints extensively for functionality.